### PR TITLE
Feature/mc 11899 hypercloud kubernetes documentation

### PR DESCRIPTION
**Pull request by m-orfanos**
#### Changes made
Common k8s files were moved to a new folder "kubernetes extension".  The hyperclouds refer to this folder for their k8s documentation.

### CODE REVIEW

**Comment by m-orfanos** [comment]
FYI @jasondias9 @Gerudhoh @y0geshchaudhary I had to restart the docker container/server for the changes to take into affect.

**Comment by m-orfanos** [comment]
@jasondias9 I will open another PR for the `cluster ID` fixes.

**Review by jasondias9 — APPROVED** [review]
LGTM Thanks for moving this!

**Review by y0geshdev — APPROVED** [review]
LGTM.

File: source/includes/kubernetes_extension/_k8_configmaps.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+| `apiVersion` <br/>_string_                 | The API version used to retrieve this configmap         |
+| `kind` <br/>_string_                       | The type of the returned resource: ConfigMap            |
+| `metadata` <br/>_object_                   | The metadata of the configmap                           |
+| `metadata.creationTimestamp` <br/>_string_ | The date of creation of the configmap as a string       |
+| `metadata.annotations` <br/>_map_          | The annotations associated to the configmap             |
+| `metadata.name` <br/>_string_              | The name of the configmap                               |
+| `metadata.namespace` <br/>_string_         | The namespace in which the configmap is created         |
+| `metadata.uid` <br/>_object_               | The UUID of the configmap                               |
+| `metadata.selfLink` <br/>_object_          | A link uniquely identifying this config map             |
+| `metadata.resourceVersion` <br/>_object_   | The version returned of the configmap                   |
```
Comment (Gerudhoh) [inline source/includes/kubernetes_extension/_k8_configmaps.md]: This text was updated in the most recent version of [Jason's PR](https://github.com/cloudops/cloudmc-api-docs/pull/169#pullrequestreview-432409315). Could you also make these changes?

**Review by Gerudhoh — CHANGES_REQUESTED** [review]

File: source/includes/kubernetes_extension/_k8_configmaps.md (hunk 1 above)
Comment (m-orfanos) [reply source/includes/kubernetes_extension/_k8_configmaps.md]: rebased 👍

Comment (Gerudhoh) [reply source/includes/kubernetes_extension/_k8_configmaps.md]: 😵  👍

Comment (m-orfanos) [reply source/includes/kubernetes_extension/_k8_configmaps.md]: Updated here 579bd276ad734ae15ca9f6343d9db7b9066f1949

**Review by Gerudhoh — APPROVED** [review]
Looks good to me! 😍

**Pull request merged**